src/components/Dragons/Dragons.jsx

```diff
@@ -0,0 +1,40 @@
+// React hooks
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+
+// Thunk Action creators
+import { fetchDragons } from '../../Redux/dragons/dragons';
+
+// Component
+import DragonItem from './DragonItem';
+
+const Dragons = () => {
+  const dragons = useSelector(state => state.dragons);
```

Comment:
- Kindly update this line too based on the previous comment as why you need to fix this errors
```
  const dragons = useSelector((state) => state.dragons);
```